app/models/concerns/metadata.rb

```diff
@@ -4,6 +4,9 @@ module Metadata
   extend ActiveSupport::Concern
 
   included do
+    property :creator_id, predicate: RDF::Vocab::SCHEMA.identifier do |index|
+      index.as :stored_searchable, :facetable
```

Comment:
we might want to `index.as: :symbol`

see https://github.com/samvera/active_fedora/blob/511ee837cd9a461021e53d5e20f362b634de39a8/lib/active_fedora/indexing/default_descriptors.rb

Reply:
Done!